client/lib/plans/constants.js

```diff
@@ -36,6 +36,8 @@ export const PLAN_HOST_BUNDLE = 'host-bundle';
 export const PLAN_WPCOM_ENTERPRISE = 'wpcom-enterprise';
 export const PLAN_CHARGEBACK = 'chargeback';
 
+export const PLAN_BUSINESS_ONBOARDING_EXPIRE = '2020-07-27T00:00:00+00:00';
```

Comment:
Small thing, but for simplicity, if this is only used in the code above where it has 1 year added to it, is there any reason not to just set this date to 2021 and get rid of the `.add( '1', 'year' )` part?

Reply:
Very good point. I changed it to 2021-07-31 so it matches 1 year from deploying the QS mention removal.